src/tests/ftest/rebuild/basic.py

```diff
@@ -25,7 +25,7 @@ def test_rebuild_basic(self):
             Multiple pool rebuild, single client, various record/object counts
 
         :avocado: tags=all,daily_regression
-        :avocado: tags=vm
+        :avocado: tags=hw,large
```

Comment:
@daltonbohning any objection to moving this to HW Large to avoid the intermittent DAOS-17796 issue